tests/test_train_model.py

```diff
@@ -1,45 +1,49 @@
+"""Test for train_model module contians parameters checks"""
 # Import packages
 import unittest
 import os
-from senbace.prediction_and_training import make_prediction 
-from senbace.prediction_and_training import train_model 
+from senbace.prediction_and_training import make_prediction
+from senbace.prediction_and_training import train_model
 
 class TestTrainModel(unittest.TestCase):
- 
+    """Contains smoke and edge tests for the parameters
+    """
     def test_smoke1(self):
-        """Smoke test: making sure model runs when all 3 parameters are given"""
+        """
+        Smoke Test: Making sure model runs when all 3 parameters are given correctly
+        """
         num_epochs = 10
         learning_rate = 0.0001
         data_batch_size = 3
         smoke_data_loader = make_prediction.load_data(os.getcwd()+"/tests/test_images/", data_batch_size)
-        accuracy_list, loss_list = train_model.train_data(num_epochs, learning_rate,data_batch_size,smoke_data_loader, "AlexNet")
```

Comment:
The train_data function now has an additional argument being passed, so the "AlexNet" argument should stay in all lines in this test